packages/lib/src/vis/shared/AxisSystemProvider.tsx

```diff
@@ -57,9 +59,13 @@ function AxisSystemProvider(props: PropsWithChildren<Props>) {
     const { width, height } = availableSize;
     return new Matrix4()
       .makeScale(width / 2, -height / 2, 1) // scale from normalized camera space to HTML space
-      .setPosition(width / 2, height / 2); // account for shift of (0,0) position (center for camera, top-left for HTML)
+      .setPosition(width / 2, height / 2, 0); // account for shift of (0,0) position (center for camera, top-left for HTML)
```

Comment:
BTW, this was giving an `undefined` value in the matrix, which led to `NaN` values when inverting the matrix.

Reply:
Well spotted :monocle_face: